airflow/models/dag.py

```diff
@@ -1767,6 +1772,24 @@ class DagModel(Base):
     def __repr__(self):
         return "<DAG: {self.dag_id}>".format(self=self)
 
+    def get_local_fileloc(self):
```

Comment:
Make sure that we use `get_local_fileloc` instead of `fileloc`

Reply:
Checked for rest of the instances, looks good